chemberta/data/pubchem_descriptors_sample_1k_normalization_values_207.json

```diff
@@ -0,0 +1 @@
7364968724171, 5.191845255002325, 5.687920668781853, 25.882901027363687, 21.81061233749132, 13.628340630239043, 12.190825095850945, 11.298419625949936, 16.0632646381433, 1.1154210739047483, 7.8019839620141465, 5.387855530416102, 28.607442701765933, 14.032999277243459, 30.976649719975487, 0.0, 8.149836061901189, 8.321995257622982, 7.268315104203722, 4.179489985997787, 10.796438519729234, 21.38844042101321, 9.567183450983887, 7.865562657778215, 25.630210817726653, 27.105664356170738, 3.862125954590409, 10.646516885206543, 0.0, 46.56658854444853, 17.22184721918502, 9.80118873468777, 0.6690910541954899, 11.880662965119551, 13.195977113746721, 16.976273528681908, 18.717371269691935, 14.667720890272815, 18.560521587003716, 23.99381739954181, 7.919845402763496, 17.472106239830367, 3.536442096150582, 17.81284403987551, 7.952601667447624, 3.110649399723042, 2.054304793852144, 8.39654185891723, 4.619782591775674, 3.8173625142017715, 2.128019209532572, 0.236056326333339, 10.439248288634497, 1.8792006968716721, 3.068913332085255, 0.6632959241579333, 0.7792521161261685, 1.0150860895207392, 1.12904925920315, 0.885841207313423, 1.3622612288234084, 2.3966890688754994, 1.461076093829663, 3.352077805601189, 4.546703459891795, 0.5812825827362633, 0.6531928468142993, 0.8703368399324923, 1.6043726028844687, 2.6706940369181096, 40.109479427800515, 0.3075924765825182, 0.39812416052223804, 0.37218127207002255, 0.29522572601379227, 0.16822178467460672, 1.3207509309708272, 0.31750830262049046, 0.1895939233353124, 0.347720719094043, 0.3550916135562478, 1.348263256862627, 1.348263256862627, 0.11754837908380168, 0.10481393483650434, 0.1831640373547582, 1.3948617612604828, 1.2289512325323424, 0.46702248542632624, 0.2115548774594604, 0.2672878863427151, 0.37142937633340856, 0.31750830262049046, 0.03162276075862127, 0.10914124661012291, 0.0891284456038114, 0.9416988407216558, 0.7968785326360313, 1.2210078745032462, 0.14753138395947027, 0.8459317261873186, 0.7140260194697382, 0.054717318793659306, 0.031622760758621284, 0.044698926049582074, 1.1304949179453359, 0.0, 1.177711604018516, 0.03162276075862126, 0.0, 0.044698926049581915, 0.4002777589954339, 0.8856975419473017, 0.20376210755180174, 0.12933459764834604, 1.2018024854771152, 0.12933459764834596, 0.12553686011284088, 0.2523952118852928, 0.2001951951326321, 0.0, 0.0, 0.30099352752196873, 0.2552375899204281, 0.054717318793659896, 0.054717318793659514, 0.52315687181949, 0.17066993679739162, 0.2074029627607446, 0.21298516379624982, 0.19953342417032294, 0.16304951303901577, 0.054717318793659785, 0.08912844560381099, 0.044698926049582095, 0.5130264408994892, 0.18710955336362645, 0.18710955336362645, 0.10943463758731921, 0.07068229607171143, 0.3691129264338621, 0.271502031286384, 0.1434543052552089, 0.0, 0.4744132278442473, 0.5481904093808613, 0.30981910814483865, 0.2969718781307522, 0.15952702685037234, 0.13746918438520642, 0.0705687957237828, 0.24861527970953778, 0.0, 0.2654559535208344, 1.6328078444361518, 0.22055666355482306]}
```

Comment:
how do we know which one of these files to use?

Reply:
yeah so I discovered that the newer `rdkit 2021.03.1` release actually added some additional descriptors; before, there were 200 and now there are 208 (in both cases, we remove `Ipc` which I found takes a while to compute).